docs/glossary.md

```diff
@@ -322,6 +381,19 @@ A [simulation network](advanced/simnet.md) with very low difficulty, such that a
 
 A wallet mode in which only blocks related to addresses owned by the wallet are downloaded. When not in SPV mode, wallets must download all blocks in the blockchain to verify transactions (fully validating mode). SPV mode allows wallets to operate with less stringent hardware requirements and load significantly faster. SPV wallets cannot vote, but can purchase tickets and allocate voting rights to a [Voting Service Provider (VSP)](#voting-service-provider).
 
+#### Smart contract
+
+A smart contract is a self-executing contract with the terms of the agreement
```

Comment:
Convention in glossary is to start with an article. I.e. this,
>A smart contract is a self-executing contract with the terms of the agreement

Would be:
> A self-executing contract with the terms of the agreement

The other new entries need to be similarly changed to adhere to convention.

Reply:
Agreed